updateinstructions.md

```diff
@@ -0,0 +1,9 @@
+# How to Update Muledump
```

Comment:
Seems fine, but maybe better to be added to README, or linked from it at the very least?